include/oneapi/dpl/pstl/tuple_impl.h

```diff
@@ -256,14 +298,49 @@ struct tuple<T1, T...>
 
     using tuple_type = ::std::tuple<T1, T...>;
 
+    template <::std::size_t I>
+    constexpr auto
```

Comment:
Discuss ``constexpr``-ness

Reply:
Why don't we need `constexpr` here? `get_impl<I>` marked as `constexpr`, `std::move` within `get_impl` also marked as `constexpr` since C++14

Reply:
>Why don't we need constexpr here?

As you can see `constexpr` is here and I think it should be there. The issue is we can mark this function as `constexpr` since C++14 only because C++11 makes `constexpr` member functions implicitly `const` that causes redifinition issue.

Reply:
So, it we can have the macro that makes it `constexpr` since C++14

Reply:
That is my plan. I just wanted to make sure there is no objections to that. And seems no.